soh/soh/Enhancements/randomizer/randomizerTypes.h

```diff
@@ -1764,3 +1764,30 @@ typedef struct CowIdentity {
     RandomizerInf randomizerInf;
     RandomizerCheck randomizerCheck;
 } CowIdentity;
+
+typedef enum {
```

Comment:
The tracker enums feels a bit ambiguous to me, maybe specify by changing it to TRACKER_WINDOW etc instead? And WINDOW_WINDOW feels weird, maybe WINDOW_WINDOWED might be clearer?

Reply:
how about this https://github.com/HarbourMasters/Shipwright/pull/2976/commits/3016fa79997df1663afed6394416ce31dfbd5cb5 (renamed all the usages in the next commit)

Reply:
Still not a fan of TRACKER_WINDOW_WINDOW but not a blocker for me either, so up to you.

Reply:
Maybe something like TRACKER_WINTYPE_XXX instead?

Reply:
`TRACKER_WINTYPE_WINDOW` feels as odd to me as `TRACKER_WINDOW_WINDOW`

I'm also not a huge fan of `WINDOWED` because that's more of like a windowed vs fullscreen thing in my mind than what imgui means by this

i'm happy with it as is for now and if we think of a better name later it's easy to change

Reply:
really this problem goes beyond the enum imo, we have
![image](https://github.com/HarbourMasters/Shipwright/assets/70942617/537e29ed-684e-4a66-a37d-5e41401b62f1)

how is "Window" a window type?

Reply:
Yeah, I was just trying to match it to that, lol. That is also very clunky. The problem is coming up with a pair of words more accurate and less confusing XD

Reply:
Seems like something we can also address later if we want to then.

Reply:
Agreed, this PR as it stands is good to go imo.

Reply:
https://github.com/HarbourMasters/Shipwright/issues/2978